hr_holidays_credit/tests/test_hr_holidays_credit.py

```diff
@@ -148,7 +148,8 @@ def test_6(self):
         name = leave_type.with_context(
             employee_id=employee.id,
         ).name_get()[0][1]
-        self.assertTrue('available + credit' in name)
+        self.assertIn('available', name, '%s not found in %s' % ('available', name))
```

Comment:
What's the point of formatting the error message?

Reply:
To know why test don't pass, So, I add messages to display compared values. I can remove it know but that's here... Let me know.
It was because my odoo test db default language was not English (add -l or --load-language to command line does not help) I add to create a new one with english as default language

Reply:
I'm not sure if that's the correct way, to include localization in testing. I'd refer this to other @OCA/human-resources-maintainers

Reply:
The goal is not to test localization, it was to undesrtand why test failed has I had no tested values in logs

Reply:
Runbot tests in en_US, so probably does not matter

Reply:
Yes but when I ran tests locally I can't see why the tests failed ( tests results didn't print variables content)
I then add messages to the tests which allowed me to see that the messages were translated and to run the tests again on a basis with English as the default language.

Reply:
I removed custom messages in the last commit in the same time a had updated the tests